crates/weaver_semconv/src/group.rs

```diff
@@ -86,6 +86,10 @@ pub struct GroupSpec {
     pub name: Option<String>,
     /// The readable name for attribute groups used when generating registry tables.
     pub display_name: Option<String>,
+    // /// The event body definition
+    // /// Note: only valid if type is event
+    // #[serde(skip_serializing_if = "Option::is_none")]
+    // pub body: Option<AnyValueSpec>,   <-- This is from my Event PR (Just showing what the change is going to be here)
```

Comment:
As comment at the end of the line says, this is what it looks like (locally) in my Events PR, uncomment this and "add" the `event.yaml` file to the test and it will (at least it does in my Events PR) "load" this into the registry -- but as thats not the point of the PR I left it commented out (there are other changes I have locally -- not much) around validation.